packages/babel-plugin-remove-graphql-queries/src/index.js

```diff
@@ -196,7 +196,7 @@ export default function ({ types: t }) {
             ) {
               const identifier = t.identifier(`staticQueryData`)
               const filename = state.file.opts.filename
-              const shortResultPath = `public/static/d/${this.queryHash}.json`
+              const shortResultPath = `public/page-data/static/d/${this.queryHash}.json`
```

Comment:
this is a breaking change, we need to do this when gatsby core tells us to do so, otherwise older gatsby version will start breaking (especially after cleaning lock files). Alternatively needs major version bump for babel pluin

Reply:
Correct. I'm weary of adding an option that gatsby core can pass in because maintaining both will be more overhead? You okay with a new major version?

Reply:
It appears that this is not necessary since new published `gatsby` will have minimum set to newly published `babel-plugin-remove-graphql-queries`

Reply:
I'm fine with major bump, but I wonder if just making location configurable through babel plugin options and defaulting to legacy one would be fine for readability and maintainability.

We don't need core and plugin (or other deps like cli etc) to be on same major version, but having them on same major feels to me like this skips additional mental overhead of which dep version is used in which core version

Reply:
Yeah, agreed. 

> but I wonder if just making location configurable through babel plugin options and defaulting to legacy one would be fine for readability and maintainability.

Okay, you have me convinced. On it.

Reply:
And done @pieh